test/api/v3/integration/tasks/PUT-tasks_id.test.js

```diff
@@ -176,6 +177,42 @@ describe('PUT /tasks/:id', () => {
       expect(savedDaily.reminders[0].id).to.equal(id1);
       expect(savedDaily.reminders[1].id).to.equal(id2);
     });
+
+    it('can set a shortName if the shortName does not already exist', async () => {
+      let savedDaily = await user.put(`/tasks/${daily._id}`, {
+        shortName: 'short-name',
+      });
+
+      expect(savedDaily.shortName).to.eql('short-name');
+    });
+
+    it('ignores shortName if it is already set', async () => {
```

Comment:
Can you explain _why_ you want that?

The way I see it, the short name is like the url of a blog post. Something that is human readable and unique to the post. If you _do_ change it, any link to the post will be broken, just like any integrations that use the short name will be broken. So we should provide warnings to that effect on the website when someone tries to change it, but not totally prevent them from doing so. 

I can see someone making a simple typo that they want to correct, and being unable to do so. I'd find that very frustrating.

If you feel very strongly about it, I'm willing to hear why, and ultimately, I'll do what you want to do in this regard, but it seems very strange to me to put that limit on it.